### ISSUE TITLE
Unable to parse file /PDepend/DependencyInjection/../../../resources\services.xml

### ISSUE THREAD

**Issue by xeagle2** [phpmd/phpmd#437]
Windows 10
PHP 5.6.28 (cli) (built: Nov  9 2016 06:40:27)

The following error is thrown when the phpmd.phar 2.6.0 is launched
Unable to parse file "phar://C:/phpmd-2.6.0.phar/vendor/pdepend/pdepend/src/main/php/PDepend/DependencyInjection/../../../resources\services.xml".

The version phpmd.phar 2.5.0 works without this issue.

**Comment by ravage84** [comment]
Thanks @xeagle2 .
Could you please report this issue to [pedepend](https://github.com/pdepend/pdepend)?

**Comment by xeagle2** [comment]
Posted to https://github.com/pdepend/pdepend/issues/296

**Comment by manuelpichler** [comment]
Has anyone else with Windows the same problem? I cannot reproduce this behavior under Linux.

**Comment by xeagle2** [comment]
I don't think it is reproducible in Linux, such kind of the issues happen basically under Windows only.

**Comment by jeblad** [comment]
A Vagrant box running Debian Jessie64 has the same problem, see pdepend/pdepend#296
